examples/pytorch/recommendation/dlrm/quantization/ptq/fx/run_benchmark.sh

```diff
@@ -52,7 +52,7 @@ function init_params {
 # run_benchmark
 function run_benchmark {
     if [[ ${mode} == "accuracy" ]]; then
-        mode_cmd=" --benchmark"
+        mode_cmd=" --accuracy_only"
         default_bs=16384
     elif [[ ${mode} == "benchmark" ]]; then
```

Comment:
![image](https://user-images.githubusercontent.com/51692656/208011703-3a4f2916-0bea-49df-a2bd-e7c0c87d9c1b.png)
please use accuracy/performance for benchmark mode.